### PR TITLE
BUG: Downloads grep for windows required to generate docstrings

### PR DESCRIPTION
**Pull request by fbudin69500**

### CODE REVIEW

File: scripts/windows-download-cache-and-build-module-wheels.ps1 [hunk 1]
```diff
@@ -6,4 +6,7 @@ Invoke-WebRequest -Uri "https://github.com/InsightSoftwareConsortium/ITKPythonBu
 sz x ITKPythonBuilds-windows.zip -oC:\P -aoa -r
 Invoke-WebRequest -Uri "http://ftp.stack.nl/pub/users/dimitri/doxygen-1.8.11.windows.bin.zip" -OutFile "doxygen-1.8.11.windows.bin.zip"
 sz x doxygen-1.8.11.windows.bin.zip -oC:\P\doxygen -aoa -r
+Invoke-WebRequest -Uri "https://midas3.kitware.com/midas/download/bitstream/462235/grep-win.zip" -OutFile "grep-win.zip"
+sz x grep-win.zip -oC:\P\grep -aoa -r
+$env:Path += ";C:\P\grep"
```
Comment (thewtex) [inline scripts/windows-download-cache-and-build-module-wheels.ps1]: `Path` -> `PATH`?

Comment (fbudin69500) [reply scripts/windows-download-cache-and-build-module-wheels.ps1]: It doesn't matter on Windows.

```PS C:\Users\francois.budin> $env:PATH
C:\P\doxygen;c:\P;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\Resources\bin;C:\WINDOWS\system32;C:\WINDOWS;C
:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\CMake\bin;C:\Program Files\doxygen\
bin;C:\cygwin64\bin;C:\Program Files\Microsoft SQL Server\110\Tools\Binn\;C:\Program Files (x86)\Microsoft SDKs\TypeScri
pt\1.0\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Tools;C:\Python36;C:\Python36\Scripts;C:\Users\francois
.budin\AppData\Local\Microsoft\WindowsApps;
PS C:\Users\francois.budin> $env:Path
C:\P\doxygen;c:\P;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\Resources\bin;C:\WINDOWS\system32;C:\WINDOWS;C
:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\CMake\bin;C:\Program Files\doxygen\
bin;C:\cygwin64\bin;C:\Program Files\Microsoft SQL Server\110\Tools\Binn\;C:\Program Files (x86)\Microsoft SDKs\TypeScri
pt\1.0\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Tools;C:\Python36;C:\Python36\Scripts;C:\Users\francois
.budin\AppData\Local\Microsoft\WindowsApps;
PS C:\Users\francois.budin> $env:PatH
C:\P\doxygen;c:\P;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\Resources\bin;C:\WINDOWS\system32;C:\WINDOWS;C
:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\CMake\bin;C:\Program Files\doxygen\
bin;C:\cygwin64\bin;C:\Program Files\Microsoft SQL Server\110\Tools\Binn\;C:\Program Files (x86)\Microsoft SDKs\TypeScri
pt\1.0\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Tools;C:\Python36;C:\Python36\Scripts;C:\Users\francois
.budin\AppData\Local\Microsoft\WindowsApps;```

Comment (fbudin69500) [reply scripts/windows-download-cache-and-build-module-wheels.ps1]: I also checked on multiple machines, and the system "path" variable was always spelled "Path"

Comment (thewtex) [reply scripts/windows-download-cache-and-build-module-wheels.ps1]: Interesting.

**Pull request merged**